frontend/src/containers/home/Home.js

```diff
@@ -1,36 +1,27 @@
-import React, { Component } from "react"; 
-import Header from "./../../components/header/Header";
-import Social from "../../components/home/social/Social";
-import Community from "../../components/home/community/Community";
-import Partners from "../../components/home/partners/Partners";
-import Events from "../../components/home/events/Events";
+import React, { Component } from 'react'
+import Header from './../../components/header/Header'
```

Comment:
As a good practice try to keep the dependencies on the top and separated by a space the app dependencies.